minetest.conf.example

```diff
@@ -1385,7 +1385,7 @@
 # ask_reconnect_on_crash = false
 
 #    From how far clients know about objects, stated in mapblocks (16 nodes).
-#    
+#
```

Comment:
Wrong file. `settingtypes.txt` should be fixed and then exported to `minetest.conf.example` with this code:
https://github.com/minetest/minetest/blob/9d3135a21b31eea6dcec67b887243bcb6124b6a1/builtin/mainmenu/dlg_settings_advanced.lua#L1105-L1110

Reply:
Or rather the code be fixed, it generates these trailing spaces.